src/components/Container.js

```diff
@@ -5,172 +5,223 @@ import Projection from 'ol/proj';
 import MapControls from './MapControls';
 import OpenLayersMap from './OpenLayersMap';
 import TrailList from './TrailList';
-import {getElevation} from '../utils/mapUtils';
-import kill_logo from './../imgs/Kill_Logo.png'
-import {Image} from 'react-bootstrap';
+import {getElevation, getMapStyle} from '../utils/mapUtils';
+import kill_logo from './../imgs/Kill_Logo.png';
```

Comment:
We can get  rid of the Killington Logo since this is going to a general tool for different resorts